src/core/services/interimElement/interimElement.spec.js

```diff
@@ -362,6 +362,21 @@ describe('$$interimElement service', function() {
         expect(autoClosed).toBe(true);
       }));
 
+      it('calls onShowing', inject(function() {
+        var onShowingStarted = false;
+        Service.show({
+          template: '<some-element />',
+          passingOptions: true,
+          onShowing: onShowing
+        });
+
+        expect(onShowingStarted).toBe(true);
+
+        function onShowing(scope, el) {
+          onShowingStarted = true;
+        }
+      }));
```

Comment:
This test asserts that onShowing is called, but not that it's called in the right order.